<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="DESKTOP-1OIHU9F" failures="1" tests="2" name="Test1" time="67.443" errors="0" timestamp="2024-10-23T07:27:50 IST">
  <testcase classname="script.Demo1" name="TestA" time="7.082"/>
  <testcase classname="script.Demo2" name="TestB" time="0.164">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at script.Demo2.TestB(Demo2.java:16)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1570)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- TestB -->
  <testcase classname="generic.BaseTest" name="@BeforeSuite initReport" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite initReport -->
  <testcase classname="generic.BaseTest" name="@AfterSuite generateReport" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite generateReport -->
</testsuite> <!-- Test1 -->
